lib/logto_client.dart

```diff
@@ -115,8 +128,10 @@ class LogtoClient {
     }
 
     _tokenStorage = TokenStorage(
-        idToken: idToken,
-        accessToken: tokenResponse.refreshToken,
-        refreshToken: tokenResponse.refreshToken);
+      idToken: idToken,
+      accessToken: tokenResponse.refreshToken,
+      refreshToken: tokenResponse.refreshToken,
+    );
+    await _tokenStorage.save();
```

Comment:
Not sure, but does dart allow an async constructor? Would it be better to merge the saving steps with the TokenStorage constructor?

Reply:
Dart does not support async constructors. But you can create a static async method that returns an instance of the containing class. I don't know if that's the most elegant solution though.